xml/hwbp_registers.xml

```diff
@@ -61,10 +65,11 @@
 
             12--14: These trigger types are available for non-standard use.
 
-            15: This trigger exists (so enumeration shouldn't terminate), but
-            is not currently available.
+            15: This trigger is disabled. In this state, \RcsrTdataTwo and
+            \RcsrTdataThree can be written with any value that is supported for
+            any of the types this trigger implements. The remaining bits in this
+            register are ignored. Other values are reserved for future use.
```

Comment:
I think that the last sentence should remain a separate paragraph since it's not related to value 15.
```suggestion
            register are ignored.
            
            Other values are reserved for future use.
```